e2e/angular-extensions/src/angular-library.test.ts

```diff
@@ -139,10 +139,12 @@ describe('Angular Package', () => {
       it('empty test to make jest happy', () => {});
 
       // These fail with pnpm due to incompatibilities with ngcc for buildable libraries.
-      if (
-        getSelectedPackageManager() !== 'pnpm' ||
-        testConfig === 'publishable'
-      ) {
+      // therefore switch to yarn
+      const previousPackageRunner = process.env.SELECTED_PM;
+      if (getSelectedPackageManager() === 'pnpm') {
+        process.env.SELECTED_PM = 'yarn';
+      }
+      if (testConfig === 'publishable') {
```

Comment:
This is not right, this test is meant to run for both buildable and publishable libraries, but only for buildable libraries we need to switch the package manager to Yarn.